sydent/threepid/bind.py

```diff
@@ -22,7 +23,7 @@
 from sydent.db.invite_tokens import JoinTokenStore
 
 from sydent.db.valsession import ThreePidValSessionStore
-from sydent.db.threepid_associations import LocalAssociationStore
+from sydent.db.threepid_associations import LocalAssociationStore, GlobalAssociationStore
```

Comment:
looks unused? don't you have a PEP8 checker which catches this?

Reply:
oops. No, we probably should do if sydent were a better maintained project.